packages/edit-site/src/components/sidebar/border-panel.js

```diff
@@ -66,23 +70,22 @@ export default function BorderPanel( {
 	getStyle,
 	setStyle,
 } ) {
-	// Border style.
-	const hasBorderStyle = useHasBorderStyleControl( { supports, name } );
-	const borderStyle = getStyle( name, 'borderStyle' );
+	const units = useCustomUnits( {
+		availableUnits: [ 'px', 'em', 'rem' ],
+	} );
 
 	// Border width.
 	const hasBorderWidth = useHasBorderWidthControl( { supports, name } );
-	const borderWidthValue = parseInt(
-		getStyle( name, 'borderWidth' ) || 0,
-		10
-	);
+	const borderWidthValue = getStyle( name, 'borderWidth' );
 
-	// Border radius.
-	const hasBorderRadius = useHasBorderRadiusControl( { supports, name } );
-	const borderRadiusValue = parseInt(
-		getStyle( name, 'borderRadius' ) || 0,
-		10
-	);
+	// Step value is maintained in state so step is appropriate for current unit
+	// even when current width value is undefined.
+	const initialStep = parseUnit( borderWidthValue )[ 1 ] === 'px' ? 1 : 0.25;
+	const [ step, setStep ] = useState( initialStep );
```

Comment:
Similar comment to above, since the UnitControl component now has its own step defaults, do we still need the step logic in this component?

Reply:
✅ Done.